File: Pizza-Delivery-System/Pizza-Delivery-System/Models/Order.cs
namespace Pizza_Delivery_System.Models
{
    public class Order : BaseModel
    {
       public int UserId { get; set; }
       public List<Pizza> OrderItem { get; set; }
       public string DeliveryAddress { get; set; }
       public DateTime OrderDate { get; set; }
       public string Status { get; set; }
       public Decimal OrderTotal { get; set; }

     public Order()
        {
            OrderItem = new List<Pizza>();

        }
    }
}

File: Pizza-Delivery-System/Client/src/app/Models/Order.ts
import { BaseModel } from "./BaseModel";
import { Pizza } from "./Pizza";


export class Order extends BaseModel {
  UserId!: number;
  OrderItem!: Array<Pizza>;
  DeliveryAddress!: string;
  OrderDate!: Date;
  Status!: string;
  OrderTotal!:number;
  constructor() {
    super();
   ;
  }

}